x-pack/test/functional_with_es_ssl/apps/uptime/alert_flyout.ts

```diff
@@ -49,10 +49,6 @@ export default ({ getPageObjects, getService }: FtrProviderContext) => {
         await alerts.setAlertInterval('11');
       });
 
-      it('can set alert throttle interval', async () => {
```

Comment:
@elastic/uptime Need code owner approval on this. This PR removes the alert throttle interval field at the top of the alert flyout and moves it into the actions form, so I've removed references to it from this test.